### ISSUE TITLE
problem with common-header.mako logging in

### ISSUE THREAD

**Issue by infinityplusb** [cloudera/hue#71]
Hi

I am building a instance with hue in Amazon AWS using Ubuntu 12.04 LTS.
I am following this guide:
http://edpflager.com/?p=1973
and but using Ubuntu and not CentOS.
Once I installed the service I could not log into http://blah:8888 getting an error saying search.Search_controller not found

Looking at the error, I noticed that it was this file producing the error:
https://github.com/cloudera/hue/blob/master/desktop/core/src/desktop/templates/common_header.mako

So I went into /tmp/tmpILkfKQ/desktop and editted the common_header.mako.py file to remove all references to search
Basically I removed all the template lines generated by lines 391-407.
I was then able to log in successfully as expected.

I can try reproduce the error if needed.

Brian
